### ISSUE TITLE
Docker issue initializing with MariaDB SQL syntax error

### ISSUE THREAD

**Issue by rclai** [metabase/metabase#5924]
I installed metabase using Kitematic and this is what happens when it initializes:

```
09-08 23:29:56 INFO metabase.db :: Verifying mysql Database Connection ...
09-08 23:29:57 INFO metabase.db :: Verify Database Connection ...  ✅
09-08 23:29:57 INFO metabase.db :: Running Database Migrations...
09-08 23:29:57 INFO metabase.db :: Setting up Liquibase...
09-08 23:29:57 INFO metabase.db :: Liquibase is ready.
09-08 23:29:57 INFO metabase.db :: Checking if Database has unrun migrations...
09-08 23:30:01 WARN liquibase :: modifyDataType will lose primary key/autoincrement/not null settings for mysql.  Use <sql> and re-specify all configuration if this is the case
09-08 23:30:01 WARN liquibase :: modifyDataType will lose primary key/autoincrement/not null settings for mysql.  Use <sql> and re-specify all configuration if this is the case
09-08 23:30:01 INFO metabase.db :: Database has unrun migrations. Waiting for migration lock to be cleared...
09-08 23:30:01 INFO metabase.db :: Migration lock is cleared. Running migrations...
09-08 23:30:02 WARN liquibase :: modifyDataType will lose primary key/autoincrement/not null settings for mysql.  Use <sql> and re-specify all configuration if this is the case
09-08 23:30:02 WARN liquibase :: modifyDataType will lose primary key/autoincrement/not null settings for mysql.  Use <sql> and re-specify all configuration if this is the case
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'rows INT NULL, description TEXT NULL, entity_name VARCHAR(254) NULL, entity_type' at line 1
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
at com.mysql.jdbc.Util.getInstance(Util.java:387)
at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:942)
at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3966)
at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3902)
at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2526)
at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2673)
at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2549)
at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1861)
at com.mysql.jdbc.PreparedStatement.executeUpdateInternal(PreparedStatement.java:2073)
at com.mysql.jdbc.PreparedStatement.executeUpdateInternal(PreparedStatement.java:2009)
at com.mysql.jdbc.PreparedStatement.executeLargeUpdate(PreparedStatement.java:5098)
at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1994)
at clojure.java.jdbc$db_do_execute_prepared_statement$fn__10867.invoke(jdbc.clj:784)
at clojure.java.jdbc$db_transaction_STAR_.invokeStatic(jdbc.clj:637)
at clojure.java.jdbc$db_transaction_STAR_.invoke(jdbc.clj:585)
at clojure.java.jdbc$db_transaction_STAR_.invokeStatic(jdbc.clj:598)
at clojure.java.jdbc$db_transaction_STAR_.invoke(jdbc.clj:585)
at clojure.java.jdbc$db_do_execute_prepared_statement.invokeStatic(jdbc.clj:783)
at clojure.java.jdbc$db_do_execute_prepared_statement.invoke(jdbc.clj:778)
at clojure.java.jdbc$db_do_prepared.invokeStatic(jdbc.clj:814)
at clojure.java.jdbc$db_do_prepared.invoke(jdbc.clj:795)
at clojure.java.jdbc$execute_BANG_$execute_helper__10910.invoke(jdbc.clj:961)
at clojure.java.jdbc$execute_BANG_.invokeStatic(jdbc.clj:963)
at clojure.java.jdbc$execute_BANG_.invoke(jdbc.clj:943)
at clojure.java.jdbc$execute_BANG_.invokeStatic(jdbc.clj:954)
at clojure.java.jdbc$execute_BANG_.invoke(jdbc.clj:943)
at metabase.db$migrate_up_if_needed_BANG_.invokeStatic(db.clj:164)
at metabase.db$migrate_up_if_needed_BANG_.invoke(db.clj:151)
at metabase.db$migrate_BANG_$fn__19465.invoke(db.clj:247)
at clojure.java.jdbc$db_transaction_STAR_.invokeStatic(jdbc.clj:613)
at clojure.java.jdbc$db_transaction_STAR_.invoke(jdbc.clj:585)
at clojure.java.jdbc$db_transaction_STAR_.invokeStatic(jdbc.clj:629)
at clojure.java.jdbc$db_transaction_STAR_.invoke(jdbc.clj:585)
at clojure.java.jdbc$db_transaction_STAR_.invokeStatic(jdbc.clj:598)
at clojure.java.jdbc$db_transaction_STAR_.invoke(jdbc.clj:585)
at metabase.db$migrate_BANG_.invokeStatic(db.clj:235)
at metabase.db$migrate_BANG_.invoke(db.clj:218)
at metabase.db$run_schema_migrations_BANG_.invokeStatic(db.clj:365)
at metabase.db$run_schema_migrations_BANG_.invoke(db.clj:360)
at metabase.db$setup_db_BANG_.invokeStatic(db.clj:383)
at metabase.db$setup_db_BANG_.doInvoke(db.clj:376)
at clojure.lang.RestFn.invoke(RestFn.java:421)
at metabase.core$init_BANG_.invokeStatic(core.clj:103)
at metabase.core$init_BANG_.invoke(core.clj:82)
at metabase.core$start_normally.invokeStatic(core.clj:187)
at metabase.core$start_normally.invoke(core.clj:181)
at metabase.core$_main.invokeStatic(core.clj:279)
at metabase.core$_main.doInvoke(core.clj:274)
at clojure.lang.RestFn.invoke(RestFn.java:397)
at clojure.lang.AFn.applyToHelper(AFn.java:152)
at clojure.lang.RestFn.applyTo(RestFn.java:132)
at metabase.core.main(Unknown Source)
09-08 23:30:59 ERROR metabase.core :: Metabase Initialization FAILED:  You have an error in your SQL
 syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near
 'rows INT NULL, description TEXT NULL, entity_name VARCHAR(254) NULL, entity_type' at line 1

09-08 23:30:59 INFO metabase.core :: Metabase Shutting Down ...
09-08 23:30:59 INFO metabase.core :: Metabase Shutdown COMPLETE
```

**Comment by salsakran** [comment]
Can you try using MySQL and seeing if that error persists? We technically don't support MariaDB, except to the degree to which it is protocol compatible with MySQL. Which version of MariaDB are you using?

**Comment by rclai** [comment]
Yeah, it worked when I switched to MySQL.

**Comment by seeekr** [comment]
Has anything changed in the meantime? I have an existing MariaDB 10.3 instance I wanted to reuse for metabase's DB, getting the same error as above, and not really keen on running an additional MySQL or Postgres instance just for metabase...

**Comment by flamber** [comment]
This has been fixed since version 0.30.0
